.github/workflows/build.yml

```diff
@@ -57,9 +57,17 @@ jobs:
       if: runner.os == 'macOS'
       run: .github/scripts/install-macos-deps.sh
 
-    - name: Setup
+    - name: Setup Linux
+      if: runner.os == 'Linux'
       run: meson setup --buildtype release build
 
+    - name: Setup macOS
+      if: runner.os == 'macOS'
+      run: meson setup --buildtype release -Daccelerate=true 
+        -Daudiotoolbox=true -Dapple_lock=true -Dcarbon=true 
+        -Dcocoa=true -Dcorefoundation=true -Dcoremidi=true 
+        -Dmach_time=true -Ddispatch=true -Dmanta=false build
```

Comment:
its not essential for this PR, but just noting that it would be nice to do the cross-platform build on macOS too. i guess it will require some rejigging of the build matrix, which will also be helpful for Nix builds which are not currently in CI.